### DISCUSSION TITLE
Can you load svg in Maui Xaml?

### DISCUSSION THREAD

**Discussion by developer9969** [dotnet/maui#3661] [General]
Hi
In xamarin forms we use SVG a lot instead of pngs using FFImageloading. Does MAUI have the ability to load SVG?

Any examples out there? I am using MAUI with XAML

Thanks

**Comment by nogginbox** [comment]
You absolutely can, but they get turned into PNGs to work on all the platforms.

In my MAUI app I have the following file:
```
/Resources/Images/nogginbox.svg
```
But in my XAML I refer to it as a png  and some auto magic happens to make that work across all the platforms:
```
<Image 
    Source="nogginbox.png"
    SemanticProperties.Description="The Nogginbox SVG image scaled to a 100x100 PNG"
    WidthRequest="100"
    HeightRequest="100" />
```
If you're using PNGs to make scaling easy then this is perfect and is working really well for me. If there is another technical reason you want SVGs, then maybe this won't be fit all your needs.

**Comment by dhaval5382** [comment]
As per above post it is a automatically  convert SVG file to PNG file. 
Is this affect on MAUI application performance? 
is it work  on different resolution? png file stratch on high resolution.
if yes so please give me suggestion how to use vector image or svg file in MAUI XAML file.
In my project, so many png file will use.
